src/components/MetadataForm/MetadataForm.tsx

```diff
@@ -7,6 +7,8 @@ import { OnChangeCallback } from '../../types';
 
 export type Props = Omit<ComponentProps<typeof JsonForms>, 'onChange'> & {
 	onChange: OnChangeCallback;
+	headerTitle?: string | null;
```

Comment:
In TypeScript the consensus seems to be to refrain from using null. Microsoft themselves suggest to only use undefined.

Reply:
Changed